indigo/analysis/refs/base.py

```diff
@@ -160,13 +160,13 @@ def setup_subtypes(self):
         self.pattern_re = re.compile(
             fr'''
                 (?P<ref>
-                    (?P<subtype>{subtypes_string})\s*
-                    (No\.?\s*)?
+                    \b(?P<subtype>{subtypes_string})\s*
+                    ([nN]o\.?\s*)?
                     (?P<num>[a-z0-9-]+)
                     (\s+of\s+|/)
```

Comment:
we might find that we actually want to support NO X OF Y (and other variations), but let's see? (if that happens we'll need a bit more tweaking because ACT etc also won't match now)

Reply:
Yes, you're right. Let's see how often that happens. In my sampling, "OF" isn't common.

This only applies to sub-types, "ACT" is handled by a different matcher.